docs/api/IoT.js-API-BLE.md

```diff
@@ -9,64 +9,42 @@ The following shows ble module APIs available for each platform.
 | ble.setServices | O | O | X |
 
 
-### Contents
+# BLE - Bluetooth Low Energy
 
-- [BLE](#ble)
-    - [Module Functions](#module-functions)
-        - [`ble.startAdvertising(name, serviceUuids[, callback])`](#blestartadvertisingname-serviceuuids-callback)
-        - [`ble.stopAdvertising(callback)`](#blestopadvertisingcallback)
-        - [`ble.stopAdvertising(callback)`](#blestopadvertisingcallback-1)
-        - [`ble.setServices(services[, callback])`](#blesetservicesservices-callback)
-    - [Events](#events)
-        - [`'stateChange'`](#statechange)
-        - [`'advertisingStart'`](#advertisingstart)
-
-
-# BLE
-
-
-## Module Functions
-
-
-### `ble.startAdvertising(name, serviceUuids[, callback])`
-* `name <String>` - maximum 26 bytes
-* `serviceUuids <Array>`
+### ble.startAdvertising(name, serviceUuids[, callback])
+* `name` {string} - maximum 26 bytes
+* `serviceUuids` {Array}
   * 1 128-bit service UUID
   * 1 128-bit service UUID + 2 16-bit service UUIDs
   * 7 16-bit service UUID
-* `callback <Function(error)>`
```

Comment:
Sorry, refers to the above line. How do you pass a 128 bit value as an array? Would be good to add description after the arguments.